pennylane/init.py

```diff
@@ -20,6 +20,70 @@
 import numpy as np
 
 
+def qaoa_embedding_uniform(n_layers, n_wires, low=0, high=2 * pi, seed=None):
+    r"""Creates a parameter array for :func:`~.QAOAEmbedding`, drawn from a uniform
+    distribution.
+
+    Each parameter is drawn uniformly at random
+    from between ``low`` and ``high``. The parameters define the the trainable angles of 'ZZ interactions' and
+    the 'local fields'.
+
+    Args:
+        n_layers (int): number of layers
+        n_wires (int): number of qubits
+        low (float): minimum value of uniform distribution
+        high (float): maximum value of uniform distribution
+        seed (int): seed used in sampling the parameters, makes function call deterministic
+
+    Returns:
+        array: parameter array
+    """
+    if seed is not None:
+        np.random.seed(seed)
+
+    if n_wires == 1:
+        shp = (n_layers, 1)
+    elif n_wires == 2:
+        shp = (n_layers, 3)
+    else:
+        shp = (n_layers, 2*n_wires)
```

Comment:
A minor suggestion here could be to organize these lines into another function (e.g. called ``get_shape()``) which could then be used in ``qaoa_embedding_uniform`` and ``qaoa_embedding_normal`` as well

Reply:
The argument against this is that the init module and templates sit in very different places, and it would be confusing to share a function...Also, this is very explicit?

Reply:
I personally prefer to keep it explicit within this template, as it can add developer overhead if a single function is split into separate parts. The only argument to moving to a external function is if the same pattern is repeated/starts to get repeated in numerous places.